pylint/checkers/strings.py

```diff
@@ -104,7 +105,11 @@
               "Used when a PEP 3101 format string uses a lookup specifier "
               "({a[1]}), but the argument passed for formatting "
               "doesn't contain or doesn't have that key as an attribute.",
-              {'minversion': (2, 7)})
+              {'minversion': (2, 7)}),
+    'W1308': ("Mixing string literals concatenation with comma-separated %s",
+              "str-literals-concatenation-and-comma",
+              "String literals are implicitly concatenated in a "
+              "comma-separated list : maybe a comma is missing ?")
```

Comment:
_comma-separated **list**_ lies when message is emitted for tuple

Reply:
I don't see any example of "variablized" detailed message.
Maybe "in a comma-separated iterable definition" instead ?